ingestors/npm.go

```diff
@@ -36,12 +34,21 @@ type NPMChangeDoc struct {
 	Time map[string]string `json:"time"`
 }
 
+func (ingestor *NPM) Name() string {
+	return "npm"
+}
+
 func (ingestor *NPM) Ingest(results chan data.PackageVersion) {
-	since, err := ingestor.GetLatestSequence()
+	since, err := getBookmark(ingestor, nil)
 	if err != nil {
 		log.WithFields(log.Fields{"ingestor": "npm"}).Fatal(err)
 	}
 
+	// TODO: This is a migration for bookmarks, can delete after first deployment, change default above to "now"
+	if since == nil {
+		since, _ = redis.Client.Get(context.Background(), "npm:updates:latest_sequence").Result()
+	}
```

Comment:
While I realize "shouldn't happen"... should we have the `if since == nil { since = "now" }` fallback here?

Reply:
Yea totally, I just skipped it since I'm literally gonna delete as soon as this deploys.

Reply:
My original thought was to do it in the redis console, but this felt safer than tying a string around my finger :D